### PR TITLE
CU-6mq222 - Make typescript extend node config

### PR DESCRIPTION
**Pull request by nsallerin**
## Description

- Make typescript configuration extend node's one.
- Add Pull request template.

## Related Issue

[CU-6mq222](https://app.clickup.com/t/6mq222)

## Motivation and Context

We don't want to choose between typescript and node config, we want both.

## How Has This Been Tested?

all existing yarn tests passed

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)
- ~Bug fix (non-breaking change which fixes an issue)~
- ~New feature (non-breaking change which adds functionality)~
- ~Breaking change (fix or feature that would cause existing functionality to 
  change)~

## Checklist:

- ✅  My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- ✅  I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
- :red_circle: I have added tests to cover my changes.
- ✅  All new and existing tests passed.

[CONTRIBUTING_FILE]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-6mq222 [eslint-config] Make the typescript config extends node's](https://app.clickup.com/t/6mq222)

**Comment by nsallerin** [comment]
@Fenntasy oh yes my mistake I forgot to update the version in it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<!--- If you're unsure about any of these, don't hesitate to ask. We're -->
+<!--- here to help! -->
+- :red_circle: My code follows the code style of this project.
+- :red_circle: My change requires a change to the documentation.
+- :red_circle: I have updated the documentation accordingly.
+- :red_circle: I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
+- :red_circle: I have added tests to cover my changes.
+- :red_circle: All new and existing tests passed.
+
+[CONTRIBUTING_FILE]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md
```
Comment (Fenntasy) [inline .github/PULL_REQUEST_TEMPLATE.md:56]: Not the right link then

Comment (nsallerin) [reply .github/PULL_REQUEST_TEMPLATE.md]: So as we discussed I removed the .github folder and we will decide about it later

File: package.json [hunk 2]
```diff
@@ -4,23 +4,23 @@
...
     "eslint-config-prettier": "6.11.0",
-    "eslint-plugin-import": "2.20.2",
-    "eslint-plugin-prettier": "3.1.1",
-    "jest": "25.4.0",
+    "eslint-plugin-import": "2.22.0",
+    "eslint-plugin-prettier": "3.1.4",
+    "jest": "26.1.0",
     "prettier": "2.0.5",
-    "typescript": "3.8.3"
+    "typescript": "3.9.7"
```
Comment (Fenntasy) [inline package.json:23]: Thanks for the bump 🙂

**Pull request merged**